src/App.js

```diff
@@ -1,11 +1,85 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect } from 'react';
 import './App.css';
+import './assets/Main.css';
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import Sidebars from './components/Sidebar';
+import { FitMeUpThunk } from './redux/fitMeUpSlice';
+import Home from './components/Home';
+import Appointment from './components/Appointment';
+import TrainerDetails from './components/TrainerDetails';
+import SearchLocation from './components/SearchLocation';
+import Speciality from './components/Speciality';
+import Trainer from './components/Trainer';
+import Login from './components/Login';
+import ProtectedRoute from './auth/ProtectedRoute';
+import SignUp from './components/SignUp';
+import AppoinmentDetails from './components/AppoinmentDetails';
+
+const App = () => {
```

Comment:
- [ ] As a new user, whenever I try to make appointments, the list won't get populated with my new appointments:

![image](https://user-images.githubusercontent.com/57421823/216668415-94c68f86-e662-4958-abba-0c4edfe00dd4.png)


According to the console, the reservations are created on the database, but the frontend is having trouble displaying them.

Kindly fix this bug so your app functions as intended for every user.